ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+_Please use this to only for bug reports_. For questions or when you need help, you can use the [GitHub Discussions](https://github.com/kroxylicious/kroxylicious/discussions)
```

Comment:
I don't think there's a way to inject a different link per repo, maybe you could use a relative link like `../discussions`? Or we could turn on discussions at the organisation level and use https://github.com/kroxylicious/discussions

Reply:
+1

Reply:
I'm on the fence as to org wide -vs- per project discussions.